dll/win32/browseui/toolsband.cpp

```diff
@@ -38,7 +38,6 @@ class CToolsBand :
 {
 private:
     CComPtr<IDockingWindowSite> fDockSite;
-    GUID                        fExecCommandCategory;
```

Comment:
Not sure if all task extension stuff is merged in?
If not, this file might cause conflicts.
Cc @yagoulas

Reply:
They are. LGTM.
We also need to remove the following line "CComPtr<IOleCommandTarget> fExecCommandTarget;"